vcluster/deploy/control-plane/binary/security/air-gapped.mdx

```diff
@@ -0,0 +1,205 @@
+---
```

Comment:
Can you look through docs issues to make sure we have everything covered: 812, 971, 871, 1004. 
For example 871 would need to adjust other page, at least to include standalone in compatible box at top, maybe edit content.